vmdb/app/models/ems_redhat.rb

```diff
@@ -19,7 +19,7 @@ def self.raw_connect(server, port, username, password, service = "Service")
     require 'ovirt'
     params = {
       :server     => server,
-      :port       => port,
+      :port       => port.to_i,
```

Comment:
Fixed.. though ongoing activity in #2243 / #2219 may yet obviate this change entirely.